qipr_approver/approver/templates/templatetags/form_input.html

```diff
@@ -1,4 +1,4 @@
 <div class="form-input-wrapper input_{{ name }} ">
     <label for="{{ name }}">{{ label }}</label>
-    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" {{ is_disabled }}/>
+    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" placeholder="{{ placeholder }}" {{ is_disabled }}/>
```

Comment:
Yeah its fine. I just wanted to make sure that if you have an input with an emptry string as the placeholder it didn't do anything weird.